compiler/noirc_frontend/src/signed_field.rs

```diff
@@ -98,6 +106,14 @@ impl SignedField {
     pub fn to_field_element(self) -> FieldElement {
         if self.is_negative { -self.field } else { self.field }
     }
+
+    pub fn to_u128(self) -> u128 {
+        self.to_field_element().to_u128()
+    }
+
+    pub fn to_i128(self) -> i128 {
+        self.to_field_element().to_i128()
```

Comment:
Does this handle negative values?

Reply:
No, but now it does. I added a couple of tests. Thanks!